app/scripts/modules/amazon/src/serverGroup/details/rollback/rollbackServerGroup.controller.js

```diff
@@ -18,6 +18,31 @@ module.exports = angular.module('spinnaker.amazon.serverGroup.details.rollback.c
 
       var desired = serverGroup.capacity.desired;
 
+      var rollbackType = 'EXPLICIT';
+
+      if (allServerGroups.length === 0 && serverGroup.entityTags) {
```

Comment:
I'm not familiar with all the use cases, but do we really want to restrict this capability to clusters where there are no other server groups? What if someone wants to roll back a couple of releases?

Reply:
The dominant use case (at least what edge/GregB thinks is 99%) is you've destroyed the previous server group.

I was looking to avoid asking the user to make a choice on what or how they wanted to rollback. I'll reach out to Ludo and GregB once it's live and get some feedback.

At least now we've got the metadata on what the previous versions were, so if someone did want to rollback a few separate server groups in the same region/cluster, they would at least be able to fairly easily determine what the previous versions were.

Reply:
🌮